recipe/meta.yaml

```diff
@@ -10,15 +10,15 @@ package:
   version: {{ version }}
 
 source:
-  url: https://ambermd.org/downloads/AmberTools23_rc6.tar.bz2
+  url: https://ambermd.org/downloads/AmberTools23.3.tar.bz2
   sha256: debb52e6ef2e1b4eaa917a8b4d4934bd2388659c660501a81ea044903bf9ee9d
   patches:
     - patches/do_not_symlink_missing_ff12pol.patch
     - patches/do_not_install_vendored_parmed.patch
     - patches/do_not_vendor_packmol.patch
 
 build:
-  number: 1
```

Comment:
This is a new build (the first of this version) so we have to drop the build number to 0. If we were making a new build of the same version (usually something like updating a version of a dependency, but not building from a different release tarball) we would have to increment this.